msvc/ut-build-early.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Starboard OneCore Overrides">
+    <_SB_Local_VCInstallDir Condition="'$(_SB_Local_VCInstallDir)' == ''">$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\14.0\Setup\VC@ProductDir)</_SB_Local_VCInstallDir>
```

Comment:
>14.0 [](start = 136, length = 4)

What will happen in the next VS version? 15.0?  
Is it possible to use /NoDefaultLib in each UT and specify the libs with some VS variable that is set in MS.Cpp.Defaults.props